ui/src/pages/help-center/help-center-card/HelpCenterCard.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button, CardContent, Typography, Card, Stack } from '@mui/material';
+import {
+  Support,
+  ForumOutlined,
+  DatasetOutlined,
+  NorthEast,
+  SaveAlt,
+  MapOutlined,
+} from '@mui/icons-material';
+import { KnowledgeBaseIcon } from 'icons';
+import { FC, ReactNode, useCallback } from 'react';
+import { CARD_IDS, START_ICON } from '../HelpCenter.constants';
+import { HelpCenterCardProps, CardButton } from './HelpCenterCard.types';
+
+export const HelpCenterCard: FC<HelpCenterCardProps> = (props) => {
+  const { shouldDisplayCard, card } = props;
+
+  const { id, title, borderColor, description, buttons, adminOnly } = card;
+
+  const getIcon = useCallback((cardId: string): ReactNode => {
+    switch (cardId) {
+      case CARD_IDS.pmmDocs:
+        return <KnowledgeBaseIcon />;
+      case CARD_IDS.support:
+        return <Support />;
+      case CARD_IDS.forum:
+        return <ForumOutlined />;
+      case CARD_IDS.pmmDump:
+        return <DatasetOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const getButtonStartIcon = useCallback((iconName: string): ReactNode => {
+    switch (iconName) {
+      case START_ICON.download:
+        return <SaveAlt />;
+      case START_ICON.map:
+        return <MapOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const onButtonClick = useCallback((button: CardButton) => {
+    if (button.target !== '') {
+      window.open(button.url, button.target, 'noopener,noreferrer');
+    } else {
+      window.location.assign(button.url);
+    }
+  }, []);
+
+  if (!shouldDisplayCard(adminOnly)) {
+    return null;
+  }
+
+  return (
+    <Card
+      sx={{
+        borderTop: borderColor ? `solid 12px ${borderColor}` : 'none',
+      }}
+      key={id}
+      data-testid={`help-card-${id}`}
+    >
+      <CardContent sx={{ paddingRight: '16px', paddingLeft: '16px' }}>
+        <Stack
+          flexDirection={'row'}
+          justifyContent={'flex-start'}
+          alignItems={'center'}
+          marginBottom={'8px'}
+        >
+          {getIcon(id)}
+          <Typography variant="h6" sx={{ marginLeft: getIcon(id) ? '8px' : '0px' }}>
```

Comment:
You can use [spacing](https://mui.com/system/spacing/#notation) and [short prop names](https://mui.com/system/properties/#properties-reference-table) from MUI for these values 

so for example `{ mr: 1 }` would get translated to `{ marginLeft: 8px }`

This also applies to other values in the component.